### PR TITLE
improve security of playground and meadow

### PR DESCRIPTION
**Pull request by shmish111**
* better sandboxing of plutus playground and meadow services
* rate limiting

### CODE REVIEW

**Comment by shmish111** [comment]
@michaelpj I promise the refactoring is coming but I all ready had this branch underway before https://github.com/input-output-hk/plutus/pull/650 was merged.

**Comment by shmish111** [comment]
Sorry about the formatting but I found a nix formatter

File: plutus-playground/deployment/nixops/meadow.nix [hunk 1]
```diff
@@ -1,98 +1,124 @@
...
+  script = "meadow  --config ${playgroundConfig} webserver -b 127.0.0.1 -p 4000 ${plutus.meadow.client}";
+}; 
 };
 }
+
+
+
+
+
+
```
Comment (michaelpj) [inline plutus-playground/deployment/nixops/meadow.nix]: Excess extra whitespace

**Comment by shmish111** [comment]
@michaelpj https://github.com/nixcloud/nix-beautify I am a big proponent of auto-formatting all code, even if you don't like the format so I would be up for that (in its own PR of course)

**Pull request merged**